script.js

```diff
@@ -65,3 +65,12 @@ function updateUncheckedTodoCount() {
   const todosLeftElement = document.getElementById("todos-left");
   todosLeftElement.textContent = `Todos left unchecked: ${uncheckedTodos}`;
 }
+
+document
+  .getElementById("clearCheckedTodosBtn")
+  .addEventListener("click", () => {
+    state.todos = state.todos.filter((todo) => !todo.checked);
+
+    renderTodos();
+    updateUncheckedTodoCount();
```

Comment:
You do not need to update the count cos this click of the button has nothing to do with check/uncheck of a todo.